src/NextGen/DonationForm/Blocks/DonationFormBlock/Block.php

```diff
@@ -172,4 +152,19 @@ public function getEnabledPaymentGateways($formId): array
 
         return $gateways;
     }
+
+    protected function convertFormBlockDataToFieldsAPI( $block ) {
+        if( $block->innerBlocks ) {
+            $section = Section::make($block->clientId);
+            if( property_exists( $block->attributes, 'title' ) ) $section->label( $block->attributes->title );
+            foreach( $block->innerBlocks as $innerBlock ) {
+                $section->append( $this->convertFormBlockDataToFieldsAPI( $innerBlock ));
+            }
+            return $section;
+        }
+
+        $field = Text::make( $block->clientId);
```

Comment:
@kjohnson is there way to dynamically determine what Field type we should use here?  I looked at the json stored in the content briefly and I don't see a property for input type on the root level of each object. I guess we could currently derive the type by the name of the block but that would be tedious.

Reply:
I'm still weighing how to map blocks to field types. Leaning towards a type attribute on the registered block. For the prototype I'm starting with assuming Text.

Reply:
@kjohnson got it, for now I just want to make sure we preserve the form functionality so I added a crude conditional 0f14278f